### PR TITLE
added some hash functions

### PR DESCRIPTION
**Pull request by michaelsmith213**

### CODE REVIEW

**Comment by michaelsmith213** [comment]
hi krishpranav i have added some hash value i think so you will added prelude headers so only i assumed prelude code and created my own hash header

**Comment by krishpranav** [comment]
hi @michaelsmith213 i will check the code

**Comment by krishpranav** [comment]
yes i have idea to add prelude headers

File: userland/libraries/libutils/Hash.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021, Krisna Pranav
```
Comment (krishpranav) [inline userland/libraries/libutils/Hash.h:2]: you should add your name here

Comment (michaelsmith213) [reply userland/libraries/libutils/Hash.h:2]: ok

**Comment by krishpranav** [comment]
**MERGED**

**Pull request merged**